src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
```

Comment:
```suggestion
    private ColorSupplier colorSupplier = new ColorSupplier();
    private Random random = ne Random();

```
рандом теж можна так закинути